### PR TITLE
Support PHP 7.3

### PR DESCRIPTION
**Pull request by mjmeijerman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Role Variables
 
 All role variables are displayed with their default value.
 
-Specify the version of PHP you want to use (`7.0`, `7.1` or `7.2`):
+Specify the version of PHP you want to use (`7.0`, `7.1`, `7.2` or `7.3`):
 
-    php7_version: 7.1
+    php7_version: 7.3
```
Comment (jaspernbrouwer) [inline README.md:20]: Do we want to switch the default to 7.3 already?

Comment (mjmeijerman) [reply README.md:20]: It is an LTS already, so why not :-)

**Review by verschoof — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mjmeijerman) [reply README.md:20]: I just noticed that I did not change the default by the way, only the default in the readme 🙈

**Review by verschoof — APPROVED** [review]

**Review by edwinkortman — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jaspernbrouwer) [reply README.md:20]: Fine by me :slightly_smiling_face: 

I do consider it a BC-break, so let's bump the major version!

**Review by jaspernbrouwer — APPROVED** [review]

**Pull request merged**